tasks/ensure_lv.json

```
@@ -80,6 +80,10 @@
     "region_size": {
       "description": "A mirror is divided into regions of this size (in MB), the mirror log uses this granularity to track which regions are in sync. CAN NOT BE CHANGED on already mirrored volume. Take your mirror size in terabytes and round up that number to the next power of 2, using that number as the -R argument.",
       "type": "Optional[Integer]"
-    } 
+    },
+    "yes_flag": {
+      "description": "Do not prompt for confirmation interactively but always assume the answer yes.",
```

```suggestion
      "description": "Do not prompt for confirmation interactively but always assume the answer false.",
```
Think we will need to update these docs to reflect the current release, then when default is set to true in the next major release we can change this back.

May be better to stick with the boolean terms (true/false) as well to avoid confusion. :)

---

Agreed! That’s much better